mitodl/mit-open
# Add certificate badge to drawer

### What are the relevant tickets?

Closes https://github.com/mitodl/hq/issues/4925

### Description (What does it do?)


- Adds the certificate badge to the learning resource drawer info section
- Aligns drawer price display with cards

### Screenshots (if appropriate):

<img width="514" alt="image" src="https://github.com/user-attachments/assets/aba18d45-950b-458b-93b2-a87edd3bff6a">

<img width="512" alt="image" src="https://github.com/user-attachments/assets/533379ca-d6ec-4099-aa6e-d52d180115d0">

### How can this be tested?


- Open the learning resource drawer (click any card)
- Confirm the certificate badge is displayed for certificated courses. The text should read:
  - e.g. "Earn a certificate: $49" for free courses with a certificate option
  - "Certificate included" for paid courses

### Additional Context

Refactors the pricing logic from ol-components/LearningResourceCard to ol-utilities so it can be shared with the drawer component (LearningResourceExpanded).
